### ISSUE TITLE
wav --> amr 

### ISSUE THREAD

**Issue by jieatgstdev** [hilen/TSVoiceConverter#6]
1.3M wav --> 127kb amr
1.3M wav time is 20s, However, transcoding amr format audio became 1m20s.
why?

**Comment by lanceylai** [comment]
Where to download the wav file?

**Comment by hupei1991** [comment]
Same issue here. The audio file is seemingly corrupted, which sounds like Lord Vader.

**Comment by lanceylai** [comment]
@hupei1991 Where can I download the audio file?

**Comment by hupei1991** [comment]
I didn't download the audio file. I just used the AudioRecorder with default setting, to record my own voice, saving as WAV file. Then I tried to convert it to AMR. Then the sound was corrupted.

**Comment by lanceylai** [comment]
Well, can you give me some infomation about the wav file. I think maybe something wrong with the audio file's detail information, such as bitstream, req_mode and etc. My E-mail is loktarlai#gmail.com. So can you send it to me? Thanks

**Comment by lanceylai** [comment]
My another project TSWechat is using the library. It works fine. Here it is: https://github.com/hilen/TSWeChat/blob/master/TSWeChat/Classes/Chat/ChatHelper/AudioRecordManager.swift#L96-L104

**Comment by hupei1991** [comment]
I agree with you concerning the header info of the audio file. Please spare me some time so that I can revert my code, and send the original and converted audio file to your email.

**Comment by hupei1991** [comment]
@hilen I have already sent the related information and supporting files to you. Please check them out and let me know if there is anything else you need.
